boxoffice/templates/boxoffice.html

```diff
@@ -0,0 +1,619 @@
+                          </p>
+                        {{/if}}
+                      {{/line_item.discount_policies}}
+                      <div class="ticket-description">
+                        <p class="valid-upto">Prices go up after {{line_item.price_valid_upto}}</p>
+                      </div>
+                      <div class="ticket-description">{{{ line_item.item_description }}}</div>
+                    </div>
+                    <div class="ticket-price-qty">
+                      <p class="ticket-price"><i class="fa fa-inr"></i> {{ line_item.base_price }}</p>
+                      <span class="multiplier">&times;</span>
+                      <div class="ticket-qty-wrapper">
+                        <button class="decrement" on-click="updateOrder(event, line_item.item_name, item.quantity_available, false)">-</button>
+                        <input type="number" class="ticket-qty" name="quantity" min="0" max="{{ item.quantity_available }}" value="{{ line_item.quantity }}" disabled>
+                        <button class="increment" on-click="updateOrder(event, line_item.item_name, item.quantity_available, true)">+</button>
+                      </div>
+                      <div class="item-amount">
+                        {{#line_item.discounted_amount && !isLoadingFail}}
+                          <p intro-outro='fly:{"x":0,"y":"0"}' class="discount-amount">Discount: <span><i class="fa fa-inr"></i> {{line_item.discounted_amount}}</span></p>
+                        {{/}}
+                        {{#line_item.discounted_amount || line_item.final_amount && !isLoadingFail}}
+                          <p intro-outro='fly:{"x":0,"y":"0"}' class="subtotal">Subtotal: <span><i class="fa fa-inr"></i> {{line_item.final_amount}}</span></p>
+                        {{/}}
+                      </div>
+                      </div>
+                    </div>
+                  {{/if}}
+                {{/}}{{/order.line_items}}
+            {{/}}{{/items}}
+            </div>
+          </div>
+        {{/section.categories}}
+        <hr>
+        <div class="ticket-total {{#loadingPrice}}grey-out{{/}} clearfix">
+          {{#loadingPrice && !isLoadingFail}}
+            <div class="loader"><i class="fa fa-spinner fa-spin fa-2x"></i></div>
+          {{/}}
+          <p class="price">Total : <span class="pull-right"><i class="fa fa-inr"></i> {{ order.final_amount }}</span></p>
+        </div>
+        {{#order.readyToCheckout}}
+          <div class="proceed-button-wrapper" intro='fly:{"x":200,"y":"0"}'>
+            <button id="stage1-proceed" class="boxoffice-button boxoffice-button-action" on-click="checkout(event)">Continue</button>
+          </div>
+        {{/order.readyToCheckout}}
+        {{#errorMsg }}
+          <p class="stage-error-msg text-danger">{{ errorMsg }}</p>
+        {{/}}
+        <div class="policy">
+          <p class="policy-heading">Cancellation and refund policy</p>
```

Comment:
Could directly link to the "cancellation and refund policy" subsection of the site policies page. Could also mention that ticket transfer is available till 24 hrs before the event.